llvm/include/llvm/Support/YAMLTraits.h

```diff
@@ -406,19 +406,19 @@ template <class T> struct has_MappingValidateTraits<T, EmptyContext> {
 
 // Test if MappingContextTraits<T>::enumInput() is defined on type T.
 template <class T, class Context> struct has_MappingEnumInputTraits {
-  using Signature_validate = void (*)(class IO &, T &);
+  using Signature_enumInput = void (*)(class IO &, T &);
```

Comment:
while we are at it, could we six up type/variable naming to conform to the llvm coding standards?